### DISCUSSION TITLE
[ONEWAY] decorator on the field in the row is not working during IN-LINE editing on the grid.

### DISCUSSION THREAD

**Discussion by javaman63** [serenity-is/Serenity#7105] [General]
I have a column declared as ONEWAY (because is a Computed Field), using the dialog works as expected.
Now, the same field can be updated from the IN-LINE editing of the grid, when I use the grid's save button it returns an error, probably because it's serializing the value and passing it to the database.
How can I make the inline editing of the grid work like the dialog, i.e., not attempt to persist the value?

I tried with Updateable(false) and Insertable(false), and with NotMapped, it doesn't seem to take into account the decorators placed on the row.

Any Ideas ?

**Comment by volkanceylan** [comment]
Inline editing is a premium feature, use StartSharp repo if you are.

If you are talking about inline editing in product grid sample in Serene, then you decide which columns are edited

**Comment by javaman63** [comment]
I'm maintaining a system that was developed using the starsharp template, but we no longer have a subscription. I suppose I could renew it in the future at my boss's discretion.
At the moment, I don't think it's (well, actually, I'm not sure) a query about a Premium Feature, but rather because it has that behavior.

I'm using a gridEditController:

`protected editController: GridEditController<PresupuestosEmpresaDetalleRow>;`

In the AfterInit:

`
afterInit() {
    super.afterInit();

    this.getGrid().onCellChange.subscribe((e, args) => {
        const { item, row } = args;
        const columnIndex = args.cell;
        
        const columnId = this.getGrid().getColumns()[columnIndex]?.id;
        var fila = args.row;
        if (columnId === 'Has' || columnId === 'RindeLys' || columnId === 'EntregaAOtrasPlantas' ||
            columnId === 'DescuentoTierra' || columnId === 'DescuentoHumedad' 
        ) {
           const producto = item.Has * item.RindeLys;
            item.Tn = producto;
            // I want to display the product in the grid, but I don't want to persist it because it's a computed field.
            this.getGrid().getData().updateItem(item.IdPresupuestoEmpresaDetalle, item); 
        }
    }
}

protected getButtons() {
    let buttons = super.getButtons();

    buttons.push(GridEditController.undoButton(() => this.editController));
    buttons.push(GridEditController.saveChangesButton(() => this.editController));

    return buttons;
}


`

When I save with the grid button (saveChange), it ignores that the field  Tn is [OneWay].
